src/javascript/app/pages/user/account/top_up_virtual.js

```diff
@@ -16,12 +17,12 @@ const TopUpVirtual = (() => {
 
         BinarySocket.send({ topup_virtual: '1' }).then((response) => {
             if (response.error) {
-                showMessage(localize(response.error.message), false);
+                showMessage(localize('You can only request additional funds if your virtual account balance falls below USD 1,000.00.'), false);
```

Comment:
should we check for a specific error code? what if error message is something general, like a server failure?

Reply:
yeah, i think we should 👍